src/browser/ReactDOM.js

```diff
@@ -84,6 +84,7 @@ var ReactDOM = mapObject({
   h6: 'h6',
   head: 'head',
   header: 'header',
+  hgroup: 'hgroup',
```

Comment:
Meh, it doesn't matter too much anymore since `React.createElement('hgroup')` works